src/app/components/table/table.scss

```diff
@@ -137,7 +138,10 @@ $tableBorderWidth: 2px;
 
       &.activeRow {
         background: rgba(43, 114, 227, 0.1);
-        @include activeRow(1);
+        @include activeRow(1, 0px 0px 0px 0px, 0px 0px 0px 0px);
+        &:last-child:not(:nth-child(-n+4)) {
+          @include activeRow(1, 0px 0px 0px 20px, 0px 0px 20px 0px);
```

Comment:
This is fine as is, but you can also pass named arguments to some or all of the parameters to sass functions which would be more readable like this:

`@include activeRow(1, $first-child-border-radius: 0px 0px 0px 20px, $last-child-border-radius: 0px 0px 20px 0px);`